packages/taquito-ledger-signer/README.md

```diff
@@ -1,10 +1,23 @@
 # Taquito Ledger Signer package
+*Documentation can be found [here](https://tezostaquito.io/docs/ledger_signer)*  
+*TypeDoc style documentation is available [here](https://tezostaquito.io/typedoc/modules/_taquito_ledger_signer.html)*
 
+## General Information
 `@taquito/ledger-signer` is an npm package that provides developers with ledger signing functionality for Taquito. It implements the Signer interface of Taquito, allowing you to sign operations from a Ledger Nano device.
 
-You first need to import the desired transport from the [LedgerJs library](https://github.com/LedgerHQ/ledgerjs). The Ledger Signer has currently been tested with `@ledgerhq/hw-transport-node-hid` for Node-based applications and with `@ledgerhq/hw-transport-u2f` for web applications.
-You can pass an instance of the transport of your choice to your Ledger Signer as follows:
+## Install
+Install the package as follows
+```
+npm install @taquito/ledger-signer
+```
+
+## Usage
+### Prerequisites
+To use the Ledger Signer we must first import the desired transport from the [LedgerJs library](https://github.com/LedgerHQ/ledgerjs).
 
+The Ledger Signer has currently been tested with `@ledgerhq/hw-transport-node-hid` for Node-based applications and with `@ledgerhq/hw-transport-u2f` for web applications.
```

Comment:
`@ledgerhq/hw-transport-u2f` is deprecated and be replace by `@ledgerhq/hw-transport-webhi`

Reply:
thank you for the headsup, will resolve.